src/vs/workbench/contrib/terminalContrib/accessibility/browser/bufferContentTracker.ts

```diff
@@ -82,7 +82,7 @@ export class BufferContentTracker {
 			const isWrapped = buffer.getLine(i + 1)?.isWrapped;
 			currentLine += line.translateToString(!isWrapped);
 			if (currentLine && !isWrapped || i === (buffer.baseY + this._xterm.raw.rows - 1)) {
-				const line = currentLine.replace(new RegExp(' ', 'g'), '\xA0');
+				const line = currentLine.replace(new RegExp('  ', 'g'), '\xA0');
```

Comment:
You want to replace 2 spaces with a space and a non-breaking space, if they're interleaved like `<space><nbsp><space><nbsp>` then it will give us the same spacing as the normal terminal while fixing the orca problem.